frontends/web/src/routes/account/send/send-wrapper.tsx

```diff
@@ -29,14 +32,40 @@ type TSendProps = {
 }
 
 export const SendWrapper = ({ accounts, code, deviceIDs, devices }: TSendProps) => {
+  const { t } = useTranslation();
   const { defaultCurrency } = useContext(RatesContext);
+
+  const [bb01Paired, setBB01Paired] = useState<boolean>();
+  const [noMobileChannelError, setNoMobileChannelError] = useState<boolean>();
+
   const account = findAccount(accounts, code);
+
+  useEffect(() => {
+    const product = deviceIDs.length > 0 ? devices[deviceIDs[0]] : undefined;
+    if (account && product === 'bitbox') {
+      const fetchData = async () => {
+        try {
+          const mobileChannel = await hasMobileChannel(deviceIDs[0])();
+          const { pairing } = await getDeviceInfo(deviceIDs[0]);
+          setBB01Paired(mobileChannel && pairing);
+          setNoMobileChannelError(pairing && !mobileChannel && isBitcoinBased(account.coinCode));
+        } catch (error) {
+          console.error(error);
+        }
+      };
+      fetchData();
+    }
+  }, [account, deviceIDs, devices]);
+
+  if (noMobileChannelError) {
+    alertUser(t('warning.sendPairing'));
+    return;
```

Comment:
Why  stop the rendering with a `return` here?

Reply:
I assume this was copied over, see my other comment. I opened a PR to fix this https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3089